src/docs/010_manual/20_install.adoc

```diff
@@ -101,42 +89,154 @@ curl -Lo dtcw.bat doctoolchain.github.io/dtcw.bat
+
+[[install-dtc-with-sdk]]
+== Install docToolchain with SDKMAN!
+
+TODO: description how to install docToolchain with https://sdkman.io/[SDKMAN!].
+
 
 [[first-command]]
 == Run your First Command
 
-To get started with docToolchain, run `tasks` as the first command:
+Call the docToolchain wrapper with `tasks --group doctoolchain` to show all tasks provided by docToolchain.
+Those tasks may be used when invoking the docToolchain wrapper script.
+
+NOTE: The first time docToolchain is called, it downloads all necessary dependencies.
+Therefore the execution of the command may take some time. Subsequent calls to docToolchain will be faster.
 
 .Linux / WSL2 with bash
 [role='primary']
 --
+
 [source, bash]
 ----
 ./dtcw tasks --group=doctoolchain
+dtcw 0.50 - 8061694f
+docToolchain 2.3.0
+Available docToolchain environments: local <.>
+Environments with docToolchain [2.3.0]: local <.>
+Using environment: local <.>
+Using Java 17.0.6 [/home/john_doe/.doctoolchain/jdk/bin/java] <.>
+Downloading https://services.gradle.org/distributions/gradle-7.5.1-bin.zip <.>
+..........10%..........20%..........30%...........40%..........50%..........60%..........70%...........80%..........90%..........100%
+
+Welcome to Gradle 7.5.1!
+
+Here are the highlights of this release:
+ - Support for Java 18
+ - Support for building with Groovy 4
+ - Much more responsive continuous builds
+ - Improved diagnostics for dependency resolution
+
+For more details see https://docs.gradle.org/7.5.1/release-notes.html
+
+To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
+Daemon will be stopped at the end of the build
+
+> Configure project :
+
+Config file '/code/docToolchainConfig.groovy' does not exist' <.>
+[ant:input]
+[ant:input] do you want me to create a default one for you? (y, n)
+y
 ----
+
+<.> List of available docToolchain environments. The output may vary depending on your system. In our example only the `local` environment is available since neither `sdk` nor `docker` was found.
+<.> Environments in which docToolchain s available. The output may vary depending on how you installed docToolchain. In our example docToolchain was found in the user's local environment in `$HOME/.doctoolchain`.
+<.> Shows the used docToolchain environment. In case docToolchain is installed in more than one environment the wrapper script picks the environment in the following order: `local`, `sdk`, and then `docker`.
+<.> Location of the used Java. In our example Java was installed in the local environment with the docToolchain wrapper script.
+<.> docToolchain was invoked the first time, thus it is downloading its dependencies.
+<.> The docToolchain configuration file `docToolchainConfig.groovy` wasn't found in the project repository. docToolchain asks if it should create a new one.
+
 --
 
 .Windows with Powershell
```

Comment:
How does the new output of `dtcw.ps1` look like?

Reply:
good point. let me fix this...

Reply:
OK, updating the documentation is the only thing missing, ATM.
Would be nice if we could close this issue since it keeps other PRs blocked.

Reply:
I currently extend the dtcw.ps1 a little bit in order to make it work with linux environments.
That lets us run tests on our linux based build servers and lets us capture the output for the documentation. I think I can finish this change today. 

Let's merge these thenges afterwards and move the docs update to another issue?

Reply:
ok. I checked the install docs file and updated what was needed to be updated. Noticed that there is no powershell output :-)